packages/grafana-ui/src/components/FadeTransition/FadeTransition.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
What about placing this component in `packages/grafana-ui/src/components/transitions/FadeTransition.tsx` instead? `transitions` would be a bucket for other animations to come in the future